api/spec/v1.yaml

```
@@ -767,17 +767,17 @@ components:
       properties:
         latest_chain_id:
           type: string
-          description: The latest chain ID at the head of indexing.
+          description: The most recently indexed chain ID.
           example: *chain_id_1
         latest_block:
           type: integer
           format: int64
-          description: The latest indexed block at the head of indexing.
+          description: The height of the most recent indexed block. Query a synced Oasis node for the latest block produced.
           example: *block_height_1
         latest_update:
           type: string
           format: date-time
-          description: The RFC 3339 formatted time of latest indexing update.
+          description: The RFC 3339 formatted time of the most recently indexed block. Compare with current time for approximate indexing progress with the Oasis Network.
```

```suggestion
          description: The RFC 3339 formatted time of the latest indexing update. Compare with current time for approximate indexing progress with the Oasis Network.
```

It's unclear to me whether "time of the recent block" refers to the time it was indexed, or the time the block was produced. I think the original phrasing is more precise.

---

Opted for "latest indexing." 🤔

---

Hrm 'latest indexing' isn't very clear to me, mainly bc I'm not sure what an 'indexing' (noun) is. I think I still prefer 'latest indexing update', wdyt?

---

We settled offline on "latest Indexer update."